github/issues_timeline.go

```diff
@@ -116,6 +117,9 @@ type Timeline struct {
 	// An object containing rename details including 'from' and 'to' attributes.
 	// Only provided for 'renamed' events.
 	Rename *Rename `json:"rename,omitempty"`
+	// The ProjectCard object is part of the starfox-preview,
```

Comment:
Let's please change this comment to describe *what* `ProjectCard` represents.
Otherwise, we are left with a code-cleanup issue when we remove the `starfox-preview` header and forget to update this comment.
A description of the struct item will also improve the resulting godocs.
Thanks.